tests/ctest/test_opts.c

```diff
@@ -0,0 +1,779 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include <kdberrors.h>
+#include <kdbopts.h>
+
+#include "tests.h"
+
+#define PROC_BASE_KEY "proc/tests/opts"
+#define SPEC_BASE_KEY "spec/tests/opts"
+
+// version 6 and 7 of clang-format don't agree whether it is supposed to be *[] or * [] so disable it here
```

Comment:
Isn't it easier if we all use the same version (which is now 6)? We can upgrade to 7 when it is in Debian stable.